kits/arm/example_impedance_control_cartesian.cpp

```diff
@@ -0,0 +1,187 @@
+/**
+In these examples we will implement various hybrid motion-force controllers using the impedance control plugin, which can be used for a wide variety of 
+applications.
+Impedance control is BEST SUITED for assigning free, rigid and springy behaviour, along/about each different axis.
+While this is perfectly useful for:
+- Having a selectively compliant end-effector,
+- Switching between fixed and free behaviour to simulate (mostly) rigid constraints, and
+- Allowing human intervention for automated operations by separating controls across different axes,
+any applications involving more salient control of the forces (as more complex functions with flexible inputs) should use our force control plugin. See ex_force_control_demoname.cpp.
+
+This comprises the following demos:
+- Fixed: A task-space pose controller implemented entirely using force control via the (PID) impedance controller.
+- Cartesian: Locks onto a particular end-effector position while having some compliant orientation.
+- Gimbal: A gimbal that locks a specific end-effector orientation, while keeping the rest of the arm compliant.
+- Floor: The end-effector is free to move but can't travel below a virtual floor. To further simulate sliding on the floor, see force_control example.
+- Damping: The end-effector behaves as 3-different damped systems (overdamped, critically damped, and underdamped), at 3 different heights.
+
+The following example is for the "Cartesian" demo:
+*/
+
+// #include "lookup.hpp"
+// #include "group.hpp"
+#include "group_command.hpp"
+#include "group_feedback.hpp"
+#include "robot_model.hpp"
+#include "arm/arm.hpp"
+#include "util/mobile_io.hpp"
+#include <chrono>
+
+using namespace hebi;
+using namespace experimental; 
+
+int main(int argc, char* argv[])
+{
+  //////////////////////////
+  ///// Arm Setup //////////
+  //////////////////////////
+
+  arm::Arm::Params params;
+
+  // Setup Module Family and Module Names
+  params.families_ = {"HEBIArm-T"};
```

Comment:
I think most of the examples, we standardize "HEBI" as the family name for consistency